app/parse.cpp

```diff
@@ -12,7 +12,7 @@ Email: {taa, dol, cau, ttt}@zurich.ibm.com
 Date: 18/08/2017
 **********************************************************/
 
-#include <utf8/utf8.h>
+#include <utf8.h>
```

Comment:
note: this is needed to aligned the include path of the manual externals with the system packages which have `utf8.h` directly in `/usr/include/utf8.h`.